webapp/src/ts/app.component.ts

```diff
@@ -92,6 +92,7 @@ export class AppComponent implements OnInit, AfterViewInit {
   reportForms;
   unreadCount = {};
   useOldActionBar = false;
+  showContent = false;
```

Comment:
Would this be better in the Store rather than directly in the component?

Also, let's call it something that explains what it represents rather than what it does, eg:

```suggestion
  initalisationComplete = false;
```